### PR TITLE
feat: Verifier scanner KIOSK and process the QR code

### LINKED ISSUE #126: Verifier scan and process the QR code

### PR DESCRIPTION
**Pull request by talwinder50**
closes #126

Signed-off-by: talwinder.kaur <talwinder.kaur@securekey.com>

### CODE REVIEW

File: pkg/restapi/rp/operation/operations.go [hunk 1]
```diff
@@ -22,7 +22,14 @@ import (
 const (
 	verifyVC            = "/verifyVC"
 	httpContentTypeJSON = "application/json"
-	vcsVerifyEndpoint   = "/verify"
+
+	// edge-service endpoint to verify presentation
```
Comment (sandrask) [inline pkg/restapi/rp/operation/operations.go]: credential

File: pkg/restapi/rp/operation/operations.go [hunk 2]
```diff
@@ -22,7 +22,14 @@ import (
...
+	vcsVerifyEndpoint = "/verify"
+
+	// edge-service endpoint to verify presentation
+	vpVerifyEndpoint = "/verifyPresentation"
+
+	verifyVP = "/verifyVP"
```
Comment (sandrask) [inline pkg/restapi/rp/operation/operations.go]: What is the difference between two?

File: pkg/restapi/rp/operation/operations_test.go [hunk 3]
```diff
@@ -141,6 +141,92 @@ func TestVerifyVC(t *testing.T) {
...
 
+func TestVerifyVP(t *testing.T) {
+	t.Run("test error from parse form", func(t *testing.T) {
+		svc := New(&Config{})
+		svc.client = &mockHTTPClient{postErr: fmt.Errorf("post error")}
+
+		rr := httptest.NewRecorder()
+		svc.verifyVP(rr, &http.Request{Method: http.MethodPost})
+		require.Equal(t, http.StatusInternalServerError, rr.Code)
+		require.Contains(t, rr.Body.String(), "failed to parse form")
```
Comment (sandrask) [inline pkg/restapi/rp/operation/operations_test.go]: Are these two tests the same?

File: pkg/restapi/rp/operation/operations.go (hunk 1 above)
Comment (talwinder50) [reply pkg/restapi/rp/operation/operations.go]: done.

File: pkg/restapi/rp/operation/operations.go (hunk 2 above)
Comment (talwinder50) [reply pkg/restapi/rp/operation/operations.go]: One was endpoint of edge-service and other was endpoint in edge sandbox ..thats how it was done for verify credentials. But i noticed its confusing there for i changed for both in edge-sandbox.

File: pkg/restapi/rp/operation/operations_test.go (hunk 3 above)
Comment (talwinder50) [reply pkg/restapi/rp/operation/operations_test.go]: No actually both are different, when is failing the test due to missing form body and other one have a invalid form body. I have added the exact error its throwing to be more specific and avoid any confusion.

**Pull request merged**